nginx-controller/nginx/templates/nginx-plus.tmpl

```diff
@@ -108,3 +108,19 @@ http {
 
     include /etc/nginx/conf.d/*.conf;
 }
+
+stream {
```

Comment:
This stream block will appear even if there are no stream snippets - is that required?